backend/app/services/runtime_work_service.py

```
@@ -2002,63 +2376,395 @@ async def _list_online_runtime_workspaces(
+def _archived_project_lookup(
+    db: Session,
+    user_id: int,
+) -> dict[tuple[str, str], dict[str, Any]]:
+    mappings = list_device_workspace_kinds(db=db, user_id=user_id)
+    project_ids = {mapping.project_id for mapping in mappings}
+    projects = (
+        db.query(Project)
+        .filter(
+            Project.user_id == user_id,
+            Project.id.in_(project_ids),
+            Project.is_active == True,
```

Ruff (0.15.18)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)